spec/functional/honour_origin_cache_headers_spec.rb

```diff
@@ -1,9 +1,66 @@
 describe 'honour_origin_cache_headers' do
+
+  let(:a_date_in_the_future) { 'Thu, 01 Dec 2015 07:00:00 GMT' }
+  let(:a_date_in_the_future_plus_one) { 'Thu, 01 Dec 2015 07:01:00 GMT' }
```

Comment:
Interestingly, if you change the current date from 2015 to 2014... the test actually just passes, no problem!